test/unit-test/CMakeLists.txt

```diff
@@ -1,6 +1,12 @@
 # Include filepaths for source and include.
 include( ${MODULE_ROOT_DIR}/sigv4FilePaths.cmake )
 
+find_package(OpenSSL REQUIRED)
+set(CMAKE_BUILD_TYPE "Debug")
+set(CMAKE_C_FLAGS "-Wall -Wextra -O0 -ggdb")
+
+add_definitions(-DNDEBUG)
```

Comment:
Probably only OpenSSL should be REQUIRED here. The rest of the flags I added just for testing things locally.

Reply:
Removed the flags